rudder-web/src/main/scala/com/normation/rudder/web/comet/AsyncDeployment.scala

```diff
@@ -170,14 +167,16 @@ class AsyncDeployment extends CometActor with CometListener with Loggable {
 
   private[this] def layout = {
 
-        <li class="dropdown">
-          <a href="#" class="dropdown-toggle"  data-toggle="dropdown">
-            <span>Status</span> <span class="badge" id="generation-status"> {statusIcon}</span><span class="caret" style="margin-left:5px"></span></a>
-          <ul class="dropdown-menu" role="menu">
-            {lastStatus}
-            <li>{currentStatus}</li>
-          </ul>
-        </li>
+    <lift:authz role="deployment_read">
```

Comment:
In fact its has changed because I removed the lift:authz in common-layout to fix the comet actor correctly and ada it back here, so I don't think it needs it specific bug (lift:authz elem are not accepted in ul html5...